pyccel/codegen/printing/cwrappercode.py

```diff
@@ -576,6 +576,7 @@ def _print_Interface(self, expr):
             # Loop for all args in every functions and create the corresponding condition and body
             for p_arg, f_arg in zip(parse_args, func.arguments):
                 collect_var,_ = self.get_PyArgParseType(used_names, f_arg, p_arg)
+                print(collect_var, type(collect_var))
```

Comment:
Don't forget to remove this before you merge

Reply:
yes I'm trying to active array tests for fortran